developer_guide.md

```diff
@@ -18,6 +18,8 @@ git submodule foreach git pull origin master
 
 ## Prerequisites
 
+You can skip the prerequisites by using the Develop with Flask steps below to call the API from a local build.  
```

Comment:
Can you move the 3 bullet point about GCP permission to a new section below like:

## (Optional) Get GCP permission

This is only needed when developing place search functionality.

- ...
- ...
- ...